### ISSUE TITLE
[1.18.2] Condensed potion weirdness (unsure of how else to explain)

### ISSUE THREAD

**Issue by unmoving-javi** [P3pp3rF1y/Reliquary#562]
So, I've been playing this for a while, finally got my apothecary stuff up and running, and I noticed something odd about condensed potions and mods that let you put potions in fluid containers (I'm using Create, for instance, but I think Immersive Engineering does too): the condensed potion shows the animation for putting something in the fluid container, but nothing is added. This even seems to occur on the apothecary cauldron itself, which is interesting.

Not sure if this is a cross-mod compat bug, or something on this end, maybe worth looking into?

I really do like the apothecary system, though, it's nice to have a way to make potions that actually stack. :)

**Comment by P3pp3rF1y** [comment]
I am not sure that I am following. I don't know how the other mods exactly treat potions, so very likely they are not compatible with Reliquary potions. But what do you mean when you refer to adding potions to apothecary cauldron? new potions are only brewed in there and there's not supposed to be a way for a potion to be put into the cauldron??

**Comment by unmoving-javi** [comment]
Oh, sorry for the confusion. I was kinda tired when I wrote this, it's not clear at all.

The issue is: when right-clicking a Reliquary potion on a fluid container (including the cauldron), it acts as if it contains liquid (plays the hand swing animation) but doesn't actually do anything. I'm assuming it's not supposed to show the animation since it's not how condensed potions work.

Really it's a super, super minor issue, but I figured it was worth reporting anyways(?)
